meta-facebook/yv35-cl/prj.conf

```diff
@@ -5,7 +5,7 @@ CONFIG_NUM_PREEMPT_PRIORITIES=56
 #CONFIG_HEAP_MEM_POOL_SIZE=16384
 #CONFIG_CMSIS_V2_MSGQ_MAX_DYNAMIC_SIZE=32768
 CONFIG_HEAP_MEM_POOL_SIZE=32768
-CONFIG_CMSIS_V2_MSGQ_MAX_DYNAMIC_SIZE=16384
+CONFIG_CMSIS_V2_MSGQ_MAX_DYNAMIC_SIZE=8000
```

Comment:
Should this be set to 8192 to be consistent with the max sizes aligning with 4K boundaries?